linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -370,6 +371,105 @@ private[emitter] object CoreJSLib {
+           *   } else {
+           *     /* Fetch or polyfill Object.getOwnPropertySymbols.
+           *      * We assume that if that function does not exist, then
+           *      * symbols do not exist at all. Therefore, the result is
+           *      * always an empty array.
+           *      */
+           *     var getOwnPropertySymbols = Object.getOwnPropertySymbols || (o => []);
+           *
+           *     // Polyfill for Reflect.ownKeys
+           *     ownKeysFun = o => Object.getOwnPropertyNames(o).concat(getOwnPropertySymbols(o));
+           *   }
+           *
+           *   // Polyfill for Object.getOwnPropertyDescriptors
+           *   return (o => {
+           *     var ownKeys = ownKeysFun(o);
+           *     var descriptors = {};
+           *     var len = ownKeys.length | 0;
+           *     var i = 0;
+           *     while (i !== len) {
+           *       var key = ownKeys[i];
+           *       /* Almost equivalent to
+           *        *   descriptors[key] = Object.getOwnPropertyDescriptor(descriptors, key);
+           *        * except that `defineProperty` will bypass any existing setter for
+           *        * the property `key` on `descriptors` or in its prototype chain.
+           *        */
+           *       Object.defineProperty(descriptors, key, {
+           *         configurable: true,
+           *         enumerable: true,
+           *         writable: true,
+           *         value: Object.getOwnPropertyDescriptor(o, key)
+           *       });
+           *       i = (i + 1) | 0;
+           *     }
+           *     return descriptors;
+           *   });
+           * })();
+           */
+          val o = varRef("o")
+          val ownKeysFun = varRef("ownKeysFun")
+          val getOwnPropertySymbols = varRef("getOwnPropertySymbols")
+          val ownKeys = varRef("ownKeys")
+          val descriptors = varRef("descriptors")
+          val len = varRef("len")
+          val i = varRef("i")
+          val key = varRef("key")
+
+          val funGenerator = genArrowFunction(Nil, Block(
+            VarDef(ownKeysFun.ident, None),
+            If((typeof(ReflectRef) !== str("undefined")) && genIdentBracketSelect(ReflectRef, "ownKeys"), {
+              ownKeysFun := genIdentBracketSelect(ReflectRef, "ownKeys")
+            }, {
+              Block(
+                  const(getOwnPropertySymbols,
+                      genIdentBracketSelect(ObjectRef, "getOwnPropertySymbols") ||
+                      genArrowFunction(paramList(o), Return(ArrayConstr(Nil)))),
+                  ownKeysFun := genArrowFunction(paramList(o), Return {
+                    Apply(
+                        genIdentBracketSelect(
+                            Apply(genIdentBracketSelect(ObjectRef, "getOwnPropertyNames"), o :: Nil),
+                            "concat"),
+                        Apply(getOwnPropertySymbols, o :: Nil) :: Nil)
+                  })
```

Comment:
Indent by 2? (also below)?

Reply:
Yes, indeed. I'll do that when I rebase on top of #4382.